src/components/floating-emojis/FloatingEmojis.js

```diff
@@ -98,7 +103,7 @@ FloatingEmojis.propTypes = {
   disableVerticalMovement: PropTypes.bool,
   distance: PropTypes.number,
   duration: PropTypes.number,
-  emoji: PropTypes.string.isRequired,
+  emoji: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
Ideally this would be renamed "emojis" (would need to be renamed in all places it is used. Also, currently the SwapDetailsState.js emojis are broken (they need to be turned into arrays)